apps/researcher/.env.development

```diff
@@ -5,3 +5,5 @@
 SEARCH_ENDPOINT_URL=https://api.colonial-heritage.triply.cc/datasets/data-hub-testing/search-graph/services/search/elasticsearch
 SPARQL_ENDPOINT_URL=https://api.colonial-heritage.triply.cc/datasets/data-hub-testing/knowledge-graph/services/kg/sparql
 DATASET_BROWSER_URL=https://dev.datasets.colonialcollections.nl
+
+COMMUNITY_LICENCE="https://creativecommons.org/licenses/by/4.0/deed.en"
```

Comment:
Should we make the scope of the license clear - that it's about enrichments provided by users?

Reply:
Renamed to `COMMUNITY_ENRICHMENT_LICENSE`